app/views/layouts/homepage.html.erb

```diff
@@ -0,0 +1 @@
+<%= render template: 'layouts/sufia' %>
```

Comment:
actually it does have some partials already. may be reasonable to copy/paste/alter it locally.